From bdf908c434c86cc2c895305245536ed115139a0d Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Thu, 14 Feb 2019 20:53:36 +0800
Subject: [PATCH 6/9] sepolicy: qcom: Rename common to vendor to avoid
 confusion

 * Remove platform specific rule from sepolicy.mk as well, it's never
   ever a valid usecase.

Change-Id: I8bf6af30407d55beb9b2c5a2afaca091564e1605
---
 qcom/sepolicy.mk                                           | 3 +--
 qcom/{common => vendor}/app.te                             | 0
 qcom/{common => vendor}/file.te                            | 0
 qcom/{common => vendor}/file_contexts                      | 0
 qcom/{common => vendor}/fsck.te                            | 0
 qcom/{common => vendor}/genfs_contexts                     | 0
 qcom/{common => vendor}/hal_lineage_livedisplay_default.te | 0
 qcom/{common => vendor}/hal_perf_default.te                | 0
 qcom/{common => vendor}/hal_power_default.te               | 0
 qcom/{common => vendor}/untrusted_app.te                   | 0
 10 files changed, 1 insertion(+), 2 deletions(-)
 rename qcom/{common => vendor}/app.te (100%)
 rename qcom/{common => vendor}/file.te (100%)
 rename qcom/{common => vendor}/file_contexts (100%)
 rename qcom/{common => vendor}/fsck.te (100%)
 rename qcom/{common => vendor}/genfs_contexts (100%)
 rename qcom/{common => vendor}/hal_lineage_livedisplay_default.te (100%)
 rename qcom/{common => vendor}/hal_perf_default.te (100%)
 rename qcom/{common => vendor}/hal_power_default.te (100%)
 rename qcom/{common => vendor}/untrusted_app.te (100%)

diff --git a/qcom/sepolicy.mk b/qcom/sepolicy.mk
index d6fbd9a..2e654f3 100644
--- a/qcom/sepolicy.mk
+++ b/qcom/sepolicy.mk
@@ -8,6 +8,5 @@ BOARD_PLAT_PRIVATE_SEPOLICY_DIR += \
 
 ifneq ($(TARGET_USES_PREBUILT_VENDOR_SEPOLICY), true)
 BOARD_SEPOLICY_DIRS += \
-    device/lineage/sepolicy/qcom/common \
-    device/lineage/sepolicy/qcom/$(TARGET_BOARD_PLATFORM)
+    device/lineage/sepolicy/qcom/vendor
 endif
diff --git a/qcom/common/app.te b/qcom/vendor/app.te
similarity index 100%
rename from qcom/common/app.te
rename to qcom/vendor/app.te
diff --git a/qcom/common/file.te b/qcom/vendor/file.te
similarity index 100%
rename from qcom/common/file.te
rename to qcom/vendor/file.te
diff --git a/qcom/common/file_contexts b/qcom/vendor/file_contexts
similarity index 100%
rename from qcom/common/file_contexts
rename to qcom/vendor/file_contexts
diff --git a/qcom/common/fsck.te b/qcom/vendor/fsck.te
similarity index 100%
rename from qcom/common/fsck.te
rename to qcom/vendor/fsck.te
diff --git a/qcom/common/genfs_contexts b/qcom/vendor/genfs_contexts
similarity index 100%
rename from qcom/common/genfs_contexts
rename to qcom/vendor/genfs_contexts
diff --git a/qcom/common/hal_lineage_livedisplay_default.te b/qcom/vendor/hal_lineage_livedisplay_default.te
similarity index 100%
rename from qcom/common/hal_lineage_livedisplay_default.te
rename to qcom/vendor/hal_lineage_livedisplay_default.te
diff --git a/qcom/common/hal_perf_default.te b/qcom/vendor/hal_perf_default.te
similarity index 100%
rename from qcom/common/hal_perf_default.te
rename to qcom/vendor/hal_perf_default.te
diff --git a/qcom/common/hal_power_default.te b/qcom/vendor/hal_power_default.te
similarity index 100%
rename from qcom/common/hal_power_default.te
rename to qcom/vendor/hal_power_default.te
diff --git a/qcom/common/untrusted_app.te b/qcom/vendor/untrusted_app.te
similarity index 100%
rename from qcom/common/untrusted_app.te
rename to qcom/vendor/untrusted_app.te
-- 
2.17.1

